src/objecttypes/api/views.py

```diff
@@ -38,6 +38,7 @@ class ObjectVersionViewSet(NestedViewSetMixin, viewsets.ModelViewSet):
     queryset = ObjectVersion.objects.order_by("object_type", "-version")
     serializer_class = ObjectVersionSerializer
     lookup_field = "version"
+    pagination_class = None
```

Comment:
I didn't add pagination to the nested resource, cause in Zaken API nested resources (zaakbesluiten, zaakeigenschappen) are not paginated. 
@joeribekker is it ok or should I paginate it?

Reply:
do not paginate nested resources

Reply:
ok, thanks!